### PR TITLE
Bug fix to image container class names

### PR DESCRIPTION
**Pull request by Quevin**
Bug fix to use drupal_html_class() on image container class names so CSS in ucsf-images-universal.css effects them properly.

### CODE REVIEW

File: ucsf_images.module [hunk 1]
```diff
@@ -41,8 +41,8 @@ function ucsf_images_entity_view_mode_info() {
   //define the standard UCSF_Images view modes
   $view_modes['file']['media_full']['label'] = 'Full';
   $view_modes['file']['half']['label'] = 'Half';
-  $view_modes['file']['original_cropped']['label'] = 'Original';
-  $view_modes['file']['media_original']['label'] = 'Raw Original';
+  $view_modes['file']['original_cropped']['label'] = 'Original Cropped';
+  $view_modes['file']['media_original']['label'] = 'Original';
```
Comment (stopfstedt) [inline ucsf_images.module:45]: this just changed last week. please verify if this reversal is indeed what you want. please see https://github.com/erykmynn/ucsf_images/commit/03cbdf2326f392d5af5ca74b4ff8adf78f3239a4

Comment (erykmynn) [inline ucsf_images.module:45]: Has this been deployed yet? This reversal is not post of the intended
changes...

Comment (stopfstedt) [inline ucsf_images.module:45]: @erykmynn thanks for the feedback. @Quevin please revert lines 44-45 to its previous state and re-commit. thanks!

Comment (stopfstedt) [inline ucsf_images.module:45]: ok, reverted the label changes.. see #5.

**Pull request merged**